.eslintrc.json

```diff
@@ -8,6 +8,23 @@
 		"curly": "off",
 		"indent": "off",
 		"no-confusing-arrow": "off",
-		"no-mixed-operators": "off"
+		"no-mixed-operators": "off",
+		"import/no-extraneous-dependencies": [
```

Comment:
The original AirBnB definition is here: https://github.com/airbnb/javascript/blob/ea8fd76c5e17abd8597baf981ad32a3ec21c48dd/packages/eslint-config-airbnb-base/rules/imports.js#L71

We probably don't need to preserve everything there, but `optionalDependencies` and `Gruntfile.js` should probably stay. (Unless we keep the comment that's currently in that file.)